bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Font.java

```diff
@@ -278,4 +312,74 @@ public static Font win32_new(Device device, long handle) {
 	return font;
 }
 
+/**
+ * Invokes platform specific functionality to allocate a new font.
+ * <p>
+ * <b>IMPORTANT:</b> This method is <em>not</em> part of the public
+ * API for <code>Font</code>. It is marked public only so that it
+ * can be shared within the packages provided by SWT. It is not
+ * available on all platforms, and should never be called from
+ * application code.
+ * </p>
+ *
+ * @param device the device on which to allocate the font
+ * @param handle the handle for the font
+ * @param zoom zoom in % of the standard resolution
+ * @return a new font object containing the specified device and handle
+ *
+ * @noreference This method is not intended to be referenced by clients.
+ * @since 3.126
+ */
+public static Font win32_new(Device device, long handle, int zoom) {
+	Font font = win32_new(device, handle);
+	font.zoom = zoom;
+	return font;
+}
+
+/**
+ * Invokes platform specific private constructor to allocate a new font.
+ * <p>
+ * <b>IMPORTANT:</b> This method is <em>not</em> part of the public
+ * API for <code>Font</code>. It is marked public only so that it
+ * can be shared within the packages provided by SWT. It is not
+ * available on all platforms, and should never be called from
+ * application code.
+ * </p>
+ *
+ * @param device the device on which to allocate the font
+ * @param fontData font data to create the font for
+ * @param zoom zoom in % of the standard resolution
+ * @return a new font object containing the specified device and handle
+ *
+ * @noreference This method is not intended to be referenced by clients.
+ * @since 3.126
+ */
+public static Font win32_new(Device device, FontData fontData, int zoom) {
+	return new Font(device, fontData, zoom);
+}
+
+/**
+ * Used to receive a font for the given zoom in the context
+ * of the current configuration of SWT at runtime.
+ * <p>
+ * <b>IMPORTANT:</b> This method is <em>not</em> part of the public
+ * API for <code>Font</code>. It is marked public only so that it
+ * can be shared within the packages provided by SWT. It is not
+ * available on all platforms, and should never be called from
+ * application code.
+ * </p>
+ *
+ * @param font font to create a font for the given target zoom
+ * @param targetZoom zoom in % of the standard resolution
+ * @return a new font object containing the specified device and handle
```

Comment:
There is no handle specified -> copy&paste mistake?

Reply:
yes, that was not correctly adapted